ingest_python_scripts/check_ingest.py

```
@@ -0,0 +1,629 @@
+}
+# Flat regex used by bash grep — covers all stages.
+ALL_STAGE_REGEX = (
+    "OaiMultiPageResponseBuilder|OaiRequestInfo|HarvestEntry|"
+    "OaiHarvester|ApiHarvester|FileHarvester|"
+    "MappingEntry|IngestRemap|EnrichEntry|JsonlEntry|"
+    "harvest complete|mapping complete|enrichment complete|jsonl complete"
+)
+
+# ---------- AWS / SSM helpers ----------
+def aws(args):
+    result = subprocess.run(["aws"] + args, capture_output=True, text=True)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)